src/main/webapp/dev/js/common/questionMsq.js

```diff
@@ -177,19 +177,18 @@ function toggleMsqMinSelectableChoices(questionNum) {
 }
 
 function toggleMsqGeneratedOptions(checkbox, questionNum) {
-    if (checkbox.checked) {
+    if ($(checkbox).prop('checked')) {
         $(`#msqChoiceTable-${questionNum}`).find('input[type=text]').prop('disabled', true);
         $(`#msqChoiceTable-${questionNum}`).hide();
         $(`#msqGenerateForSelect-${questionNum}`).prop('disabled', false);
         $(`#msqOtherOptionFlag-${questionNum}`).closest('.checkbox').hide();
-        $(`#generatedOptions-${questionNum}`).attr('value',
-                $(`#msqGenerateForSelect-${questionNum}`).prop('value'));
+        $(`#msqGeneratedOptions-${questionNum}`).val($(`#msqGenerateForSelect-${questionNum}`).prop('value'));
```

Comment:
Reuse `changeMsqGenerateFor`?
See if we need `adjustMinMaxSelectableChoices` or not (I think we need it).

Reply:
Yes we need it.

But since I am now reusing `changeMsqGenerateFor(questionNum)`, it already has the `adjustMinMaxSelectableChoices(questionNum);` invoked inside the function.